launch.go

```diff
@@ -507,24 +513,24 @@ func main() {
 		storeURL := c.String("store-uri")
 		shellBin := c.String("shell-bin")
 		buildID, err := strconv.Atoi(c.Args().Get(0))
+		buildTimeout := c.Int("build-timeout")
 
-		if err != nil {
-			return cli.ShowAppHelp(c)
-		}
+		fmt.Printf("err: %v", err)
 
-		api, err := screwdriver.New(url, token)
 		if err != nil {
-			log.Printf("Error creating Screwdriver API %v: %v", buildID, err)
-			exit(screwdriver.Failure, buildID, nil, metaSpace)
+			return cli.ShowAppHelp(c)
 		}
 
-		defer recoverPanic(buildID, api, metaSpace)
-
-		launchAction(api, buildID, workspace, emitterPath, metaSpace, storeURL, shellBin)
+		fmt.Printf("%v\n", url)
```

Comment:
Did you mean to leave all these print statements here?

Reply:
Good catch! Amended.